guides/doc-Content_Management_Guide/topics/Importing_Content.adoc

```diff
@@ -451,8 +451,11 @@ Use this option if you have one of the following errors:
 . In the {Project} web UI, navigate to *Content* > *Products*.
 . Select the product containing the corrupted repository.
 . Select the name of a repository you want to synchronize.
-. From the *Select Action* menu, select *Advanced Sync*.
-. Select the option and click *Sync*.
+. To verify the checksum, click *Verify Content Checksum* from the *Action* menu.
+. To perform optimised sync and complete sync, select *Advanced Sync*, perform the following:
+.. Select *Advanced Sync* from the *Select Action* menu.
```

Comment:
seems like a duplicate of the above line?

Reply:
Yes, it is the same as above for which you have a suggestion.

Reply:
Then you don't need to say it twice.